services/checks/checks_test.go

```diff
@@ -47,6 +51,63 @@ func TestDownloadChecks(t *testing.T) {
 	})
 }
 
+func TestLoadLocalChecks(t *testing.T) {
+	s := New(nil, nil, nil, "2.5.0")
+
+	checks, err := s.loadLocalChecks("../../testdata/checks/checks.yml")
+	require.NoError(t, err)
+	require.Len(t, checks, 3)
+
+	c1, c2, c3 := checks[0], checks[1], checks[2]
+
+	assert.Equal(t, check.PostgreSQLSelect, c1.Type)
+	assert.Equal(t, "good_check_pg", c1.Name)
+	assert.Equal(t, uint32(1), c1.Version)
+	assert.Equal(t, "rolpassword FROM pg_authid WHERE rolcanlogin", c1.Query)
+
+	assert.Equal(t, check.MySQLShow, c2.Type)
+	assert.Equal(t, "bad_check_mysql", c2.Name)
+	assert.Equal(t, uint32(1), c2.Version)
+	assert.Equal(t, "VARIABLES LIKE 'version%'", c2.Query)
+
+	assert.Equal(t, check.MongoDBBuildInfo, c3.Type)
+	assert.Equal(t, "good_check_mongo", c3.Name)
+	assert.Equal(t, uint32(1), c3.Version)
+	assert.Empty(t, c3.Query)
+}
+
+func TestCollectChecks(t *testing.T) {
+	t.Run("collect local checks", func(t *testing.T) {
+		err := os.Setenv("PERCONA_TEST_CHECKS_FILE", "../../testdata/checks/checks.yml")
+		require.NoError(t, err)
+		defer os.Unsetenv("PERCONA_TEST_CHECKS_FILE")
```

Comment:
**[golangci-lint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Error return value of `os.Unsetenv` is not checked (errcheck)

Reply:
Seems that reviewdog uses Nurlans token

Reply:
@BupycHuk ^